tests/conftest.py

```diff
@@ -263,6 +263,8 @@ def mock_env(monkeypatch, request, mock_get_secret):
     with mock.patch.dict(os.environ, clear=True):
         monkeypatch.setenv("AZURE_STORAGE_ACCOUNT", "test-storage-account")
         monkeypatch.setenv("AZURE_STORAGE_CONTAINER", "test-storage-container")
+        monkeypatch.setenv("AZURE_STORAGE_RESOURCE_GROUP", "test-storage-rg")
```

Comment:
we need tests where these env vars are missing to confirm the script doesn't crash. this is a common reason people are unable to use the scripts after updates

Reply:
Yup, added a test to consume the variable in test_blob_manager.